{
	"exceptions": {
		"auth": {
      "no_token_provided": "No token provided",
      "invalid_token": "Invalid token"
		},
    "session": {
      "not_found": "Session not found"
    },
		"user": {
			"not_found": "User not found",
			"already_exists": "User already exists"
		},
		"unexpected_error": "An unexpected error has occurred"
	}
}